src/sympc/session/__init__.py

```diff
@@ -1,6 +1,14 @@
 """Session class and utility functions used in conjunction with the session."""
+# stdlib
+from typing import Dict
 
 from .session import Session
 from .session_manager import SessionManager
+from .session_utils import get_session
+from .session_utils import set_session
 
-__all__ = ["Session", "SessionManager"]
+# Mapping for rank -> Session
+# rank is needed in the case we have VirtualMachines since we share the same workspace
+CURRENT_SESSION: Dict[int, Session] = {}
```

Comment:
Q: The methods in session_util map `uuid string` to Session objects,here the type annotation sspecified rank(int)